pages/nakshatra.vue

```diff
@@ -0,0 +1,379 @@
+<template>
+  <section class="center">
+    <b-loading :active.sync="isLoading" />
+    <form class="nakshatra-form p-4 mt-10 mb-10 max-w-xl card center is-align-content-center">
+      <div class="form-inputs content">
+      <div class="boy">
+        <h1> Boy Details</h1>
+        <label class="label text-gray-600">Rasi</label>
+        <select v-model="rasiBoy" @change="calcBoy()" class="rasiBoy card mb-5">
+        <option disabled value="0">Please select one</option>
+        <option value="1">Mesh</option>
+        <option value="2">Vrishabha</option>
+        <option value="3">Mithuna</option>
+        <option value="4">Kataka</option>
+        <option value="5">Simha</option>
+        <option value="6">Kanya</option>
+        <option value="7">Thula</option>
+        <option value="8">Vrichika</option>
+        <option value="9">Dhanus</option>
+        <option value="10">Makar</option>
+        <option value="11">Kumbh</option>
+        <option value="12">Meena</option>
+        </select>
+        <br>
+        <label class="label text-gray-600">Nakshatra</label>
+        <select v-model="nakshatraBoy" name="NakshatraValBoy" @change="shouldDisable()" class="card">
+        <option value="1">{{nakBoy1}}</option>
+        <option value="2">{{nakBoy2}}</option>
+        <option value="3">{{nakBoy3}}</option>
+        </select>
+      </div>
+      <div class="girl is-align-content-center">
+        <h1> Girl Details</h1>
+        <label class="label text-gray-600">Rasi</label>
+        <select v-model="rasiGirl" @change="calcGirl()" class="card mb-5">
+        <option disabled value="0">Please select one</option>
+        <option value="1">Mesh</option>
```

Comment:
It should `Mesha` and not `Mesh`. Use rasi names from the [spreadsheet](https://docs.google.com/spreadsheets/d/1CVQ2NFECogziuI6OHfWk3A5v_qlcvAUV_8kDzu76Ip8/edit#gid=758986764).

Reply:
Declare all Rasi names as JSON array ouside of this file and loop through instead of hardcoding . See [previewVideos variable](https://github.com/erajasekar/astrosoft-ui/blob/master/mixins/CoursePreview.ts) as eg